samples/snippets/translate_v3beta1_batch_translate_document.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3beta1_batch_translate_document]
+
+from google.cloud import translate_v3beta1 as translate
+
+
+def batch_translate_document(
+    input_uri: str,
+    output_uri: str,
+    project_id: str,
+    timeout=180,
+):
+
+    '''Batch Translating Documents'''
+
+    client = translate.TranslationServiceClient()
+
+    """The ``global`` location is not supported for batch translation"""
+    location = "us-central1"
+    # Supported file types: https://cloud.google.com/translate/docs/supported-formats
+    gcs_source = {"input_uri": input_uri}
```

Comment:
I would also mention that this must be point to a document on Google Cloud Storage.